services/core-api/app/api/mines/response_models.py

```diff
@@ -137,6 +147,7 @@ def format(self, value):
         'is_archived': fields.Boolean,
         'archived_date': fields.String,
         'archived_by': fields.String,
+        'mine_document_bundle': fields.Nested(MINE_DOCUMENT_BUNDLE_MODEL),
```

Comment:
I can see sending back the geomark_id, but I don't think any of the other fields are going to be used at all on the FE. I think I'd still rather do a fetch to get the geomark_id, as it's only used when we're displaying that particular bundle (one at a time).